src/main/java/org/folio/dew/repository/SFTPObjectStorageRepository.java

```diff
@@ -87,7 +87,7 @@ public boolean upload(String username, String password, String host, int port, S
       return true;
     } catch (Exception e) {
       log.info("Error uploading the file", e);
-      return false;
+      throw new EdifactException(String.format("Unable to upload to %s:%d%s", host, port, folder));
```

Comment:
@grabsefx what do you think if we add exception message  to thrown EdifactException?

Reply:
As I know  this message will be displayed on UI so can get some details was went wrong for user